frontend/sw360-portlet/src/main/webapp/html/projects/licenseInfo.jsp

```diff
@@ -1,5 +1,5 @@
 <%--
-  ~ Copyright Siemens AG, 2016. Part of the SW360 Portal Project.
+  ~ Copyright Siemens AG, 2017. Part of the SW360 Portal Project.
```

Comment:
please extend copyright dates instead of overwriting. e.g. '2016-2017' (applies to all files)